src/Sulu/Bundle/SecurityBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,25 @@
+sulu_security.reset_password.email:
+    path: reset/email
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:sendEmail
+
+sulu_security.reset_password.email.resend:
+    path: reset/email/resend
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:sendEmail
+        generateNewKey: false
+
+sulu_security.reset_password.reset:
+    path: reset
+    defaults:
+        _controller: SuluSecurityBundle:Resetting:reset
+
+sulu_security.contexts:
+    type: rest
+    name_prefix: sulu_security.
+    resource: Sulu\Bundle\SecurityBundle\Controller\ContextsController
+
+sulu_security.profile:
```

Comment:
is there a reason why this route is not defined in the `routing_api.yml` file?

Reply:
I think we've decided to do so, because the profile (and the contexts) are not really a REST API (profile depends on the logged in user, and therefore is not an addressable resource). So we've decided that these two routes are especially for Sulu, and said that is not part of the "API"